### ISSUE TITLE
Can we support eraser tool like paint in this editor?

### ISSUE THREAD

**Issue by akjaiswal161191** [scaleflex/filerobot-image-editor#78]
I am looking for erasing the image content using an eraser tool.Are we planning to add it in future release?

**Comment by dzmitry-stramavus** [comment]
hey @akjaiswal161191 , 

Can you provide us with usecase? And what kind of tool you mean by "eraser"? what it should do exactly?

**Comment by akjaiswal161191** [comment]
I am looking for a tool that will allow me to erase image pixels.
So if your image has some content and you want to remove it, it should doable with a freehand tool.

**Comment by akjaiswal161191** [comment]
This is useful when I want to remove the unneeded part of the image content.
With crop functionality, you can do that but it's not freehand tool.

**Comment by dzmitry-stramavus** [comment]
yes, it's on our roadmap to add freehand tool. But there is no ETA for it for now.

**Comment by akjaiswal161191** [comment]
okay, thanks. We can check if we can build that tool on top of the existing lib.

**Comment by AhmeeedMostafa** [comment]
@dmnro123 not yet, time is a bit tight currently for planning and implementing but we will give an interest soon.

**Comment by jiutian2012007** [comment]
hi  @akjaiswal161191 you can take a look at this. https://konvajs.org/docs/sandbox/Free_Drawing.html
useAnnotation({ globalCompositeOperation: mode === 'brush' ? 'source-over' : 'destination-out',  name: TOOLS_IDS.PEN, })
can work well

**Comment by SamHeijder2002** [comment]
Is there a update on the eraser?
